dubbo-rpc/dubbo-rpc-rest/src/main/java/org/apache/dubbo/rpc/protocol/rest/RestProtocol.java

```diff
@@ -113,6 +122,31 @@ public void afterUnExport() {
         return exporter;
     }
 
+    private void checkJsonCompatibility(Class<?> clazz, String jsonCheckLevel) throws RpcException {
+
+        if (jsonCheckLevel == null || JSON_CHECK_LEVEL_WARN.equals(jsonCheckLevel)) {
+            boolean compatibility = JsonCompatibilityUtil.checkClassCompatibility(clazz);
+            if (!compatibility) {
+                List<String> unsupportedMethods = JsonCompatibilityUtil.getUnsupportedMethods(clazz);
+                assert unsupportedMethods != null;
+                logger.warn("", "", "", String.format("Interface %s does not support json serialization, the specific methods are %s.", clazz.getName(), unsupportedMethods));
+            } else {
+                logger.info("Check json compatibility complete, all methods can be serialized using json.");
```

Comment:
The log level should be adjusted to debug, and the check class should be added to the log message.

Reply:
done